src/utils/AccurateMassSearch.cpp

```diff
@@ -95,12 +95,12 @@ class TOPPAccurateMassSearch :
 
     void registerOptionsAndFlags_()
     {
-        registerInputFile_("in", "<file>", "", "featureXML or consensusXML file");
+        registerInputFileList_("in", "<file>", StringList(), "featureXML or consensusXML file(s); a list can contain mixed file types");
         setValidFormats_("in", ListUtils::create<String>("featureXML,consensusXML"));
-        registerOutputFile_("out", "<file>", "", "mzTab file");
+        registerOutputFileList_("out", "<file>", StringList(), "mzTab file(s)");
```

Comment:
hmm... This is most certainly bad for KNIME node generation and will probably break KNIME workflows. Why do multiple mzTab files need to be generated?
@aiche